source/documentation/using_the_api/datatypes.md

```diff
@@ -0,0 +1,26 @@
+## Datatypes
+
+The simplest way to find a field’s datatype is to use the record in the appropriate field register. This is of the form `https://field.{domain}/records/{field}`, where `{domain}` is the [domain of your register from its base URL](#find-the-base-url-for-register-s
```

Comment:
Currently, the only field register is https://field.register.gov.uk/ so wouldn't it be better to remove the variability of `domain`?

Reply:
I think @michaelabenyohai was accounting for `alpha.openregister.org` as well. If we only have one domain it would make the documentation considerably easier to write.